sdk/cosmos/azure-cosmos/azure/cosmos/aio/_cosmos_client.py

```diff
@@ -224,7 +225,18 @@ async def create_database(  # pylint: disable=redefined-builtin
         response_hook = kwargs.pop('response_hook', None)
         offer_throughput = kwargs.pop('offer_throughput', None)
         if offer_throughput is not None:
-            request_options["offerThroughput"] = offer_throughput
+            try:
+                if offer_throughput.auto_scale_max_throughput:
+                    request_options['autoUpgradePolicy'] = _stringify_auto_scale(offer=offer_throughput)
+                elif offer_throughput.auto_scale_increment_percent:
+                    raise ValueError("auto_scale_max_throughput must be supplied in "
+                                     "conjunction with auto_scale_increment_percent")
```

Comment:
It doesn't look like we are using `auto_scale_increment_percent` in the above logic?

Reply:
the elif statement is just for if we are only supplying the `auto_scale_increment_percent` if we only supplied that then it would raise the error. However if we supplied the `auto_scale_max_throughput` we would be able to supply both and everything would work as expected.